frontend/simple/views/components/Chatroom/Message.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="c-message is-flex" :class="[variant, isSameSender && 'sameSender']">
+    <avatar :src="avatar"
+      hasMargin
+      size="sm"
+      class="c-avatar level-left"
+      :class="{ 'alignToText': !hasWhoInvisible }"
+      aria-hidden="true"
+    />
+    <div class="c-body">
+      <span class="has-text-text-light is-size-7 c-who" :class="{ 'sr-only': hasWhoInvisible }">
+        {{who}}
+      </span>
+      <!-- NOTE: Use v-html to have messages formatted (links and so on) -->
+      <p class="c-text" v-html="text" v-if="text" />
```

Comment:
From [the docs](https://vuejs.org/v2/api/#v-html):

> Dynamically rendering arbitrary HTML on your website can be very dangerous because it can easily lead to [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting). Only use `v-html` on trusted content and **never** on user-provided content.

So this has to be removed. There's a reason Slack only allows markdown formatted text. :)

Also, please check the entire project for use of `v-html` where the input is user-provided and remove it there as well. I recommend also creating a comment on that same line with a link to an issue about this (to add formatting support via a markdown parser or something).

Reply:
Oh thats a nice catch, updated the code and opened an issue for it #502